ThunderCloud/Quiz.swift

```diff
@@ -26,6 +26,12 @@ public struct QuizConfiguration {
     /// If `true` then blended learning style overrides will be applied, for example
     /// in `CollectionListItem`
     public var isBlendedLearningEnabled: Bool = false
+
+    /// If `true` then  max number of questions for quiz is set
+    public var hasMaxNumberOfQuestions: Bool = false
```

Comment:
As mentioned in Bitbucket, this can be made computed off the nullibility of `maxNumberOfQuestions`